opae.spec

```diff
@@ -120,10 +124,19 @@ pushd %{_topdir}/BUILD/%{name}-%{version}-%{opae_release}/python/pacsign
 %{__python3} setup.py install --single-version-externally-managed --root=%{buildroot} 
 popd
 
-for file in %{buildroot}%{python3_sitelib}/opae/admin/tools/{fpgaflash,fpgaotsu,fpgaport,fpgasupdate,ihex2ipmi,rsu,super_rsu,bitstream_info}.py; do
+# Make rpmlint happy about install permissions
+# admin tools
+for file in %{buildroot}%{python3_sitelib}/opae/admin/tools/{fpgaflash,fpgaotsu,fpgaport,fpgasupdate,ihex2ipmi,rsu,super_rsu,bitstream_info,opaevfio,pci_device}.py; do
```

Comment:
We're wondering why we need to treat these scripts here in opae.spec, since they are part of the opae.admin package. (and handled by its setup.py)

Reply:
fedora wise, to use opae.admin as-is would mean an new project which is pretty heavy weight.
why not break have an opae-admin-* subproject in opae.spec ?

Reply:
Doesn't `rpmlint` complain about these just because the start with a shebang? The python setup.py installation takes care of installing the entrypoints in `/usr/bin` with the correct permissions....